test/run_all_tests.sh

```diff
@@ -164,20 +168,21 @@ if [ 0 -eq ${all_tests_result} ]; then
 
   # Run coverage if available
   if [ $(which lcov) ]; then
-    rm mock-*.gc?? *_Mock.gc?? *test.gc?? \
-    && gcov --version \
+    rm mock-*.gc?? *_Mock.gc?? *test.gc??
```

Comment:
I was failing here locally and that was causing the rest of the coverage commands below to not happen. I separated out this `rm` step from the rest of the steps below so that if the `rm` fails, we still do the other steps.

Reply:
I think we can just add the `-rf` flags to the `rm` to ensure `rm` doesn't "fail".

Reply:
Good catch!

I think we can just add the `-f` flag to the `rm` to ensure `rm` doesn't "fail".

I agree it's probably better to run on it's own, because running coverage should not be dependent on it's success. Let's go ahead and add the `-f` parameter, because it will clean up the logs.